tests/swagger_generation/test_optional_converters.py

```diff
@@ -0,0 +1,37 @@
+import mock
+import pytest
+import unittest
+from importlib import reload
+
+
+# HAAAAACKS - using importlib.reload will invalidate pre-existing imports in other test modules,
+# even when imported "as" something else..  so we'll just use pytest-order to ensure this test always runs LAST.
+@pytest.mark.order(-1)
+class TestOptionalConverters(unittest.TestCase):
+    def test_optional_enum_converter(self):
+        import flask_rebar.swagger_generation.marshmallow_to_swagger as _m_to_s
```

Comment:
Note: I had to import the entire module (vs just `ALL_CONVERTERS` in order to use `importlib.reload` - which was the only way I could find/think of to reload the module with `marshmallow-enum` stuff mocked out.
Definitely open to suggestions for less nasty ways to accomplish what I did here